### PR TITLE
Hide remaining buttons

### PR DESCRIPTION
**Pull request by elmerv**
- [x] Save app
- [x] Invite Member
- [x] Transfer App
- [x] Clone App

### CODE REVIEW

**Comment by elmerv** [comment]
@vgkids I need a bit more clarification on Create App and Clone App. What permissions would prevent a user from creating a new contract or app?

File: src/components/OrganizationSettings.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <div class="page-inset">
       <h2>{{ organization.name }} | Users And Roles</h2>
-      <div class="action-row">
+      <div class="action-row" v-if = "currentUserRole === 'admin'">
```
[the remark this answers is not in the document]
Comment (elmerv) [reply src/components/OrganizationSettings.vue:6]: Yes you are right. Will update this.

**Comment by vgkids** [comment]
> @vgkids I need a bit more clarification on Create App and Clone App. What permissions would prevent a user from creating a new contract or app?

Sorry– you're right about Create App, since it gets created under their own User.

The concern with Clone is that, if the user doesn't have Update permissions on an app, they shouldn't be able to clone it either, since it would be adding data to the Org. The most they could do is make a copy, owned by themselves instead of the Org.

**Comment by elmerv** [comment]
First transfer is to an org where user is an admin, second transfer is to an org user is a member.
https://github.com/solid-adventure/trivial-ui/assets/20259244/6463191d-a8d7-40cd-8019-f5222a9230e2

**Comment by vgkids** [comment]
Closing as stale

**Pull request closed without merging**